src/components/flat-table/flat-table-cell/flat-table-cell.spec.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import FlatTableCell from './flat-table-cell.component';
+import { testStyledSystemSpacing } from '../../../__spec_helper__/test-utils';
+
+describe('FlatTableCell', () => {
+  testStyledSystemSpacing(props => <FlatTableCell { ...props } />, { py: '10px', px: 3 }, null, { modifier: ' > div' });
```

Comment:
Here's another `10px`, I wonder if it should be `8px` in line with the theme multiplier?